benchmarker/modules/do_pytorch.py

```diff
@@ -119,6 +119,7 @@ def run_internal(self):
             for epoch in range(1, self.params["nb_epoch"] + 1):
                 self.inference(model, device)
         end = timer()
-        self.params["time"] = (end - start) / self.params["nb_epoch"]
+        self.params["total_time"] = (end - start)
```

Comment:
let's call it time_total
not a bit difference, but we already have time_batch and time_apoch ;)